Assets/Scripts/Entities/BasicEntity.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BasicEntity : MonoBehaviour
+{
+    public Rigidbody2D RigidBody;
+    public Collider2D Collider;
```

Comment:
You can also use RequireComponent GetComponent<Rigidbody2D>() for example? This way you don't have to drag.

https://docs.unity3d.com/ScriptReference/RequireComponent.html has an example